frame_test.go

```diff
@@ -12,7 +12,7 @@ func TestFrame(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, [8]int{384, 192, 192, 0, 0, 0, 0, 0}, f1.Linesize())
 	require.Equal(t, int64(60928), f1.PktDts())
-	require.Equal(t, f1.c, f1.UnsafeTypedPointer())
+	require.NotNil(t, f1.UnsafePointer())
```

Comment:
Does it work if you use `require.Equal(t, unsafe.Pointer(f1.c), f1.UnsafePointer())` instead? 🤔

Reply:
Fixed and all tests passed